src/stack.js

```diff
@@ -1,5 +1,49 @@
-'use strict'
-
 export default class Stack {
-  // your code here
+  constructor() {
+    this.commissary = [],
+    this.size = 0,
+    this.topIndex = null
+  }
+
+  beefUp( item ) {
+    this.commissary[this.size] = item
+    this.topIndex = this.size
+    this.size++
+  }
+
+  beefUpMany( ...items ) {
+    items.forEach( item => {
+      this.beefUp( item )
+    })
+  }
+
+  amputatePiece() {
+    if (this.size > 0) {
+      this.size--
+      this.size > 0 ? this.topIndex = this.size - 1 : this.topIndex = null
```

Comment:
this could be rewritten as
` this.topIndex = this.size > 0 ? this.size - 1 : null`